data.json

```diff
@@ -497,6 +497,12 @@
     "imageUrl": "https://user-images.githubusercontent.com/66197682/105803643-eaf33180-5fc3-11eb-8f97-42de7cee0f05.png",
     "issueNumber": 279
   },
+  {
```

Comment:
Please Add yourself to the bottom

Reply:
Name = Rishabh Dwivedi
id= rishabh062
issue =390

Reply:
Done